### PR TITLE
feat(ci)!: support testing on OH 3.2, remove testing on 3.1

### PR DESCRIPTION
**Pull request by boc-tothefuture**

### CODE REVIEW

**Comment by boc-tothefuture** [comment]
Build looks to be failing with same issue in #438

**Comment by jimtng** [comment]
ultimately we need to fix #448 and modify the addon to import the ruby class, e.g. https://github.com/boc-tothefuture/openhab-jruby/pull/448#issuecomment-985964964, and then fix openhab-jruby to work with the modified addon.

**Comment by ccutrer** [comment]
> ultimately we need to fix #448 and modify the addon to import the ruby class, e.g. [#448 (comment)](https://github.com/boc-tothefuture/openhab-jruby/pull/448#issuecomment-985964964), and then fix openhab-jruby to work with the modified addon.

I know I know, that's my cue... Between new job and holiday stuff, I just haven't had time. Depending on how holiday break takes shape next week I'll either have some to do it, or _not_ have any time at all.

File: docs/installation/index.md [hunk 1]
```diff
@@ -13,23 +13,26 @@ has_children: false
 
 ## Installation
 
-1. Create directory for Ruby Gems `<OPENHAB_CONF>/automation/lib/ruby/gem_home`
-2. Configure JRuby OpenHAB services
+### From the user interface
+1. Install the jrubyscripting addon following the [OpenHAB instructions](https://www.openhab.org/docs/
+configuration/addons.html)
```
Comment (jimtng) [inline docs/installation/index.md:18]:
```suggestion
1. Go to Settings -> Add-ons -> Automation and install the jrubyscripting automation addon following the [OpenHAB instructions](https://www.openhab.org/docs/
configuration/addons.html) 
```

File: features/step_definitions/openhab.rb [hunk 2]
```diff
@@ -183,6 +183,7 @@ def nil_if_blank(str)
 
 Given('feature {string} installed') do |feature|
   openhab_client("feature:install #{feature}")
+  sleep 10 # Version 3.2 of openhab seems to take a while to isntall features TODO, this should be a wait_until
```
Comment (jimtng) [inline features/step_definitions/openhab.rb:186]:
```suggestion
  sleep 10 # Version 3.2 of openhab seems to take a while to install features TODO, this should be a wait_until
```

**Pull request merged**